tests/epyccel/test_higherorder_functions.py

```diff
@@ -1,70 +1,81 @@
  # pylint: disable=missing-function-docstring, missing-module-docstring/
+import pytest
 import modules.highorder_functions as mod
 from pyccel.epyccel import epyccel
 
+@pytest.mark.c
 def test_int_1():
     modnew = epyccel(mod, language = "c", verbose = True)
 
     x_expected = mod.test_int_1()
     x = modnew.test_int_1()
     assert x == x_expected
```

Comment:
Given that all these tests could be run with Fortran in the near future, wouldn't it be better to have `language` as a parameter, and set it equal to `c` using a fixture? We could also have a `fortran` option with `pytest.mark.skip`.